bud.config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Client assets build configuration
+ *
+ * @typedef {import('@roots/bud').Framework} Config
+ *
+ * @param {Config} config
+ * @returns {Config}
+ */
+module.exports = (config) =>
+  config
+    /**
+     * Use sage preset
+     */
+    .use(require('@roots/sage'))
+
+    /**
+     * Application entrypoints
+     */
+    .entry({
+      app: '**/app.{js,css}',
+      editor: '**/editor.{js,css}',
+      customizer: '**/customizer.{js,css}',
```

Comment:
I think we should be more specific here for the entries instead of using globs. It’s nice that they do double-duty for getting both the js and css assets, and they are more terse to write, but I think they are less clear for Sage users who might not understand all that is happening.

```suggestion
      app: ['resources/scripts/app.js', 'resources/styles/app.css'],
      editor: ['resources/scripts/editor.js', 'resources/styles/editor.css'],
      customizer: 'resources/scripts/customizer.js',
```